recipes/perfetto/all/CMakeLists.txt

```diff
@@ -18,6 +18,10 @@ install(
   PUBLIC_HEADER DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
 )
 
+if (PERFETTO_DISABLE_LOGGING)
```

Comment:
Use positive flags instead of negative. Negative names can cause prone errors, after all, we are not machines (yet).

Reply:
@uilianries, I actually kept it the way Google set this define, as the logging is on by default for perfetto build. This setting must be only used for projects which explicitly want to disable it. But if we want to do this other way around - I can change it.